client/src/index.js

```diff
@@ -1,20 +1,18 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
+import ReactDOM from 'react-dom/client';
 import reportWebVitals from './reportWebVitals';
 import { BrowserRouter } from 'react-router-dom';
+import App from './App';
+import './index.css';
+
+const root = ReactDOM.createRoot( document.getElementById( "root" ) );
```

Comment:
What are the advantages of the new codebase ?

Reply:
ReactDOM.render is no longer supported in React 18!
For more information read this [Deprecation Notice](https://stackoverflow.com/questions/71668256/deprecation-notice-reactdom-render-is-no-longer-supported-in-react-18)